labelbox/schema/consensus_settings.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import BaseModel
+
+
+class ConsensusSettings(BaseModel):
```

Comment:
A few things that I'd like to recommend here. @msokoloff1, would love your 2 cents here too.

- `numberOfLabels`and `coveragePercentage` should be snake_case to stay consistent
- In order to easily convert them to camelCase for graphql query, you can inherit `_CamelCaseMixin`, which currently exists in `schema.data_row_metadata`. This mixin also inherits from pydantic `BaseModel`. However, I would be keen to moving the mixin class to `labelbox.utils.py`, since we already have camelCase converters in that file.
- Once you inherit from _CamelCaseMixin, you can convert to camelCase when constructing the dictionary like so: `consensus_settings.dict(by_alias=True)`